### PR TITLE
ENH: Add webhook output to search actions

### PR DESCRIPTION
**Pull request by gmatthews20**
### Description:

Add to_webhook output which sends the results of a search action to a webhook. 
This will allow the output of a search action to send triggers for each item in the results to a webhook which will create a trigger to start other actions.

### Special Notes:

---

### Submitter:

Have you (where applicable):

* [x] Added unit tests?
* [x] Checked the latest commit runs on Dev?
* [ ] Updated the example config file(s) and README?

---

### Reviewer

Does this PR:

* [ ] Place non-StackStorm code into the `lib` directory?
* [ ] Have unit tests for the action/sensor and `lib` layers?
* [ ] Have clear and obvious action parameter names and descriptions?

### CODE REVIEW

File: lib/workflows/to_webhook.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import json
+import os
+from typing import Dict, List
+import requests
+
+
+def to_webhook(webhook: str, payload: List[Dict]) -> None:
```
Comment (meoflynn) [inline lib/workflows/to_webhook.py:7]: Could you add a docstring to this method?

File: tests/lib/workflows/test_to_webhook.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import requests
+from workflows.to_webhook import to_webhook
+
+
+@patch.dict(
+    os.environ,
+    {"ST2_ACTION_API_URL": "http://example.com", "ST2_ACTION_AUTH_TOKEN": "fake_token"},
+)
+@patch("requests.post")
+def test_to_webhook_success(mock_post):
```
Comment (meoflynn) [inline tests/lib/workflows/test_to_webhook.py:15]: Could you add a docstring to this method?

File: tests/lib/workflows/test_to_webhook.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        verify=False,
+    )
+
+
+@patch.dict(
+    os.environ,
+    {"ST2_ACTION_API_URL": "http://example.com", "ST2_ACTION_AUTH_TOKEN": "fake_token"},
+)
+@patch("workflows.to_webhook.requests.post")
+def test_to_webhook_failure(mock_post):
```
Comment (meoflynn) [inline tests/lib/workflows/test_to_webhook.py:47]: Could you add a docstring to this method?

**Review by meoflynn — CHANGES_REQUESTED** [review]

**Review by meoflynn — APPROVED** [review]

**Review by anish-mudaraddi — APPROVED** [review]

**Comment by anish-mudaraddi** [comment]
eventually every action should have a webook - but this is fine for now

**Pull request merged**